shopping_mall/backend/app/models/faq_doc.py

```
@@ -0,0 +1,141 @@
+            base["faq_category_id"] = self.faq_category_id
+
+        # 관계가 로드된 경우 슬러그/이름 포함 (검색 필터용)
+        # 세션 분리 상태를 방어적으로 처리 — lazy load 오류 대비
+        try:
+            cat = self.faq_category  # may trigger lazy load or raise DetachedInstanceError
+            if cat is not None:
+                base["subcategory_slug"] = cat.slug
+                base["subcategory_name"] = cat.name
+        except Exception:
+            # 세션 분리 상태 — 슬러그 없이 저장 (ChromaDB fallback 검색 가능)
+            pass
```

Ruff (0.15.12)

[error] `try`-`except`-`pass` detected, consider logging the exception

(S110)